spec/features/publish/courses/gcse_equivalency_spec.rb

```diff
@@ -0,0 +1,175 @@
+    @course2 ||= provider.courses.find_by(course_code: "XXX2")
+  end
+
+  def course3
+    @course3 ||= provider.courses.find_by(course_code: "XXX3")
+  end
+
+  def when_i_visit_the_course_page(course:)
+    publish_provider_courses_show_page.load(
+      provider_code: provider.provider_code, recruitment_cycle_year: provider.recruitment_cycle_year, course_code: course.course_code,
+    )
+  end
+
+  def then_i_see_course_3_gcse_requirements
+    expect(page).to have_content("Grade 4 (C) or above in English and maths")
+    expect(page).to have_content("Candidates with pending GCSEs will be considered")
+    expect(page).to have_content("Equivalency tests will be accepted in English or maths")
+    expect(page).to have_content("Cycling Proficiency")
+  end
+
+  def then_i_see_course_2_gcse_requirements
+    expect(page).to have_content("Grade 4 (C) or above in English, maths and science")
+    expect(page).to have_content("Candidates with pending GCSEs will not be considered")
+    expect(page).to have_content("Equivalency tests will not be accepted")
+  end
+
+  def then_i_see_the_form_pre_populated
+    expect(gcse_requirements_page.pending_gcse_yes_radio).to be_checked
+    expect(gcse_requirements_page.gcse_equivalency_yes_radio).to be_checked
+    expect(gcse_requirements_page.english_equivalency).to be_checked
+    expect(gcse_requirements_page.maths_equivalency).to be_checked
+    expect(gcse_requirements_page.additional_requirements).to have_content("Cycling Proficiency")
+  end
+
+  def and_i_click_save
+    gcse_requirements_page.save.click
+  end
+
+  def and_i_see_pending_gcse_and_equivalency_tests_errors
+    expect(page).to have_content("Select if you consider candidates with pending GCSEs")
+    expect(page).to have_content("Select if you consider candidates with pending equivalency tests")
+  end
+
+  def and_i_set_the_gcse_requirements
+    gcse_requirements_page.pending_gcse_yes_radio.click
+    gcse_requirements_page.gcse_equivalency_yes_radio.click
+  end
+
+  def and_there_is_no_science_equivalency
+    expect(gcse_requirements_page).not_to have_science_equivalency
+  end
+
+  def and_i_see_equivalency_errors
+    expect(page).to have_content("Enter details about equivalency tests")
+    expect(page).to have_content("Select if you accept equivalency tests in English or maths")
+  end
+
+  def then_i_fill_the_equivalency_requirements
+    gcse_requirements_page.english_equivalency.check
+    gcse_requirements_page.maths_equivalency.check
+    gcse_requirements_page.additional_requirements.set("Cycling Proficiency")
+  end
+
+  def and_i_am_on_the_course_page
+    expect(page).to have_current_path publish_provider_recruitment_cycle_course_path(
+      provider.provider_code,
+      course.recruitment_cycle.year,
+      course.course_code,
+    )
+  end
+
+  def and_i_see_the_success_summary
+    expect(publish_provider_courses_index_page.success_summary).to have_content("Success\nYour changes have been saved.")
```

Comment:
```suggestion
 expect(publish_provider_courses_index_page.success_summary).to have_content(I18n.t("success.saved"))
```

Just to save time if we ever need to change later